--- libpurple/mediamanager.c	2fafe3d7f9ea11fc9275a9114cbba9d93576c9bf
+++ libpurple/mediamanager.c	1d55fd9e6e4068c2343d62595b03f26663215867
@@ -780,7 +780,8 @@ purple_media_manager_remove_output_windo
 		pad = gst_element_get_static_pad(queue, "sink");
 		peer = gst_pad_get_peer(pad);
 		gst_object_unref(pad);
-		gst_element_release_request_pad(GST_ELEMENT_PARENT(peer), peer);
+		if (peer != NULL)
+			gst_element_release_request_pad(GST_ELEMENT_PARENT(peer), peer);
 		gst_element_set_locked_state(queue, TRUE);
 		gst_element_set_state(queue, GST_STATE_NULL);
 		gst_bin_remove(GST_BIN(GST_ELEMENT_PARENT(queue)), queue);
